app/views/articles/show.html.erb

```diff
@@ -81,143 +81,165 @@
+        </div>
+
+        <% if has_long_markdown && @collection %>
+          <%= render "articles/collection",
+                     rendered_article: @article,
+                     collection: @collection,
+                     articles: @collection_articles,
+                     position: :bottom %>
+        <% end %>
+      </article>
+    </div>
+
+    <% if has_long_markdown || @organization %>
+      <section class="crayons-card crayons-card--secondary text-padding">
+        <% if has_long_markdown %>
+          <% cache("article-about-author-#{@user.id}-#{@user.updated_at}", expires_in: 100.hours) do %>
+            <%= render "articles/about_author", published: @article.readable_publish_date %>
+          <% end %>
+        <% end %>
+        <%= render "articles/org_branding", organization: @organization, author: has_long_markdown %>
+      </section>
+    <% end %>
+
+    <div class="crayons-card text-padding">
+      <%= render "articles/full_comment_area" %>
+    </div>
+
+    <% cache("article-bottom-content-#{@article.id}", expires_in: 18.hours) do %>
+      <% @classic_article = Suggester::Articles::Classic.new(@article, not_ids: [@article.id]).get.first %>
+      <% if @classic_article %>
+        <%= render "additional_content_boxes/article_box",
+                   article: @classic_article,
+                   classification: "classic",
+                   classification_text: "Classic #{community_name} Post from #{@classic_article.readable_publish_date}",
+                   follow_cue: "Follow <a href='#{@classic_article.user.path}'>@#{@classic_article.user.username}</a> to see more of their posts in your feed." %>
       <% end %>
+      <div id="additional-content-area" class="gap-4" data-article-id="<%= @article.id %>,<%= @classic_article&.id %>"></div>
+      <% suggested_articles = ArticleSuggester.new(@article).articles(max: 4) %>
+      <%= render "articles/bottom_content", articles: suggested_articles %>
     <% end %>
-    <%= render "articles/org_branding", organization: @organization %>
-    <%= render "articles/full_comment_area" %>
   </div>
-  <% cache("article-bottom-content-#{@article.id}", expires_in: 18.hours) do %>
-    <% @classic_article = Suggester::Articles::Classic.new(@article, not_ids: [@article.id]).get.first %>
-    <% if @classic_article %>
-      <%= render "additional_content_boxes/article_box",
-                 article: @classic_article,
-                 classification: "classic",
-                 classification_text: "Classic #{community_name} Post from #{@classic_article.readable_publish_date}",
-                 follow_cue: "Follow <a href='#{@classic_article.user.path}'>@#{@classic_article.user.username}</a> to see more of their posts in your feed." %>
+
+  <div class="crayons-layout__sidebar-right">
+    <% stick_nav_cache_key = "sticky-sidebar-#{@article.id}-#{(@organization || @user).profile_updated_at}-#{(@organization || @user).last_article_at}-0-#{@user.pro?}" %>
```

Comment:
```suggestion
    <% stick_nav_cache_key = "sticky-sidebar-#{@article.id}-#{(@organization || @user).profile_updated_at}-#{(@organization || @user).last_article_at}-1-#{@user.pro?}" %>
```

Reply:
Just out of curiousity @benhalpern, what's the reason for changing part of the cache key from `0` to `1`? Is it simply a way to invalidate the cache for the sticky nav by creating a new cache key?